WooCommerce/src/main/kotlin/com/woocommerce/android/AppPrefs.kt

```diff
@@ -167,6 +175,18 @@ object AppPrefs {
         setDeletableInt(UndeletablePrefKey.CANCELLED_APP_VERSION_CODE, versionCode)
     }
 
+    fun getFCMToken(): String {
+        return getString(DeletableSitePrefKey.WC_PREF_NOTIFICATIONS_TOKEN)
+    }
```

Comment:
Nitpick, no need to change. But for getter methods like this would be nice to have this sintax: 

```suggestion
    fun getFCMToken() = getString(DeletableSitePrefKey.WC_PREF_NOTIFICATIONS_TOKEN)
```

Basically as you did for `AppPrefsWrapper.kt`

Reply:
Thank you @JorgeMucientes 🙇 , I agree and I prefer the expression body syntax, I'll update it 👍

I'd do that even for the setter and the remove function, but those could get a bit confusing; for the getters It's very clear tho` :)

Reply:
Updated 🎉